File: Quick-annotations/src/main/java/com/wpf/app/quick/annotations/BindD2VHelper.java
package com.wpf.app.quick.annotations;

//import android.view.View;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

/**
 * Created by 王朋飞 on 2022/7/6.
 */
public interface BindD2VHelper<VH, V, Data> {

    void initView(@Nullable VH viewHolder, @NonNull V view, @NonNull Data data);
}

File: QuickBind/src/main/java/com/wpf/app/quickbind/annotations/BindD2VHelper.kt
package com.wpf.app.quickbind.annotations

import androidx.recyclerview.widget.RecyclerView
import com.wpf.app.quick.annotations.BindD2VHHelper

/**
 * Created by 王朋飞 on 2022/9/5.
 *
 */
interface BindD2VHelper<View : Any, Data : Any> :
    BindD2VHHelper<RecyclerView.ViewHolder, View, Data> {

    override fun initView(viewHolder: RecyclerView.ViewHolder?, view: View, data: Data) {
        initView(view, data)
    }

    fun initView(view: View, data: Data)
}